developer/src/kmc/src/commands/buildClasses/BuildProject.ts

```diff
@@ -43,7 +44,9 @@ class ProjectBuilder {
 
     // Give a hint if the project is v1.0
     if(this.project.options.version != '2.0') {
-      this.callbacks.reportMessage(InfrastructureMessages.Hint_ProjectIsVersion10());
+      if(getOption("prompt to upgrade projects", true)) {
```

Comment:
Could you combine the if clauses?

Reply:
Yeah but I don't think it's worth it :grin: